<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/ic_receipt_black_48dp.png</normaloff>:/icons/ic_receipt_black_48dp.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="ResultWidget" name="resultWidget" native="true"/>
    </item>
    <item>
     <widget class="FiltersWidget" name="filtersWidget" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="findPushButton">
        <property name="text">
         <string>Find</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/magnifier.png</normaloff>:/icons/magnifier.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuSrvice">
    <property name="title">
     <string>&amp;Service</string>
    </property>
    <addaction name="actionLog"/>
    <addaction name="actionSQL_Log"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>F&amp;ilters</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionExpandAll"/>
    <addaction name="actionCollapseAll"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>&amp;Reports</string>
    </property>
    <addaction name="actionCreateReport"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilters"/>
   <addaction name="menuSrvice"/>
   <addaction name="menuReports"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/information-white.png</normaloff>:/icons/information-white.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>Show about dialog</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/door-open.png</normaloff>:/icons/door-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/wrench-screwdriver.png</normaloff>:/icons/wrench-screwdriver.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionSQL_Log">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/bug.png</normaloff>:/icons/bug.png</iconset>
   </property>
   <property name="text">
    <string>SQL &amp;Log</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/qt.png</normaloff>:/icons/qt.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="toolTip">
    <string>Show about Qt dialog</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/magnifier.png</normaloff>:/icons/magnifier.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/eraser.png</normaloff>:/icons/eraser.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/folder-horizontal-open.png</normaloff>:/icons/folder-horizontal-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/disk-black.png</normaloff>:/icons/disk-black.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/bug.png</normaloff>:/icons/bug.png</iconset>
   </property>
   <property name="text">
    <string>Main Log</string>
   </property>
  </action>
  <action name="actionExpandAll">
   <property name="text">
    <string>Expand All</string>
   </property>
   <property name="toolTip">
    <string>Expand all filters</string>
   </property>
  </action>
  <action name="actionCollapseAll">
   <property name="text">
    <string>Collapse All</string>
   </property>
   <property name="toolTip">
    <string>Collapse all filters</string>
   </property>
  </action>
  <action name="actionCreateReport">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/table-insert-row.png</normaloff>:/icons/table-insert-row.png</iconset>
   </property>
   <property name="text">
    <string>Report Wisard</string>
   </property>
   <property name="toolTip">
    <string>Show report wisard</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FiltersWidget</class>
   <extends>QWidget</extends>
   <header>src/filterswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResultWidget</class>
   <extends>QWidget</extends>
   <header>src/resultwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findPushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionFind</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
